modules/presets/index.js

```diff
@@ -66,6 +66,14 @@ export function presetIndex() {
         fileFetcher.get('preset_fields')
       ])
       .then(vals => {
+        // https://github.com/ideditor/schema-builder/issues/100
+        const scheduleFields = ['opening_hours', 'opening_hours/covid19', 'service_times', 'collection_times'];
+        scheduleFields.forEach(id => {
+          if (id in vals[3]) {
+            vals[3][id].type = 'schedule';
```

Comment:
Since https://github.com/ideditor/schema-builder/pull/101 is now merged, this temporary override can be removed, and then this PR would be ready to review&merge

Reply:
We’re still waiting on openstreetmap/id-tagging-schema#1537, right?